tla/consensus/ccfraft.tla

```diff
@@ -815,22 +817,22 @@ RejectAppendEntriesRequest(i, j, m, logOk) ==
        \/ /\ m.term >= currentTerm[i]
           /\ state[i] = Follower
           /\ ~logOk
-          /\ LET prevTerm == IF m.prevLogIndex = 0 THEN 0
-                             ELSE IF m.prevLogIndex > Len(log[i]) THEN 0 ELSE log[i][m.prevLogIndex].term
+          /\ LET prevTerm == IF m.prevLogIndex = 0 THEN 2
+                             ELSE IF m.prevLogIndex > Len(log[i]) THEN 2 ELSE log[i][m.prevLogIndex].term
```

Comment:
That does not seem right, in the ELSE case, something like log[i][Len(log[i])].term.

Reply:
Introduced in https://github.com/microsoft/CCF/commit/5301f6739fd19c89cef582bff998c32d49095cd2